common/utils/src/main/scala/org/apache/spark/util/LogUtils.scala

```diff
@@ -29,9 +29,9 @@ object LogUtils {
   /**
    * Schema for structured Spark logs.
    * Example usage:
-   *   val logDf = spark.read.schema(LOG_SCHEMA).json("path/to/logs")
+   *   val logDf = spark.read.schema(SPARK_LOG_SCHEMA).json("path/to/logs")
    */
-  val LOG_SCHEMA: String = """
+  val SPARK_LOG_SCHEMA: String = """
```

Comment:
Well, may I ask what kind of confusions you have in your mind? This is already `org.apache.spark.util.LogUtils.LOG_SCHEMA`, isn't it?

Reply:
After the change, there is no need to import `org.apache.spark.util.LogUtils.LOG_SCHEMA` in repl shell. If we are going to expose this by default in `bin/spark-shell` and `bin/pyspark`, we had better to make it more specific.